plugins/node.d/snmp__if_

```diff
@@ -125,9 +125,21 @@ if (defined $ARGV[0] and $ARGV[0] eq "snmpconf") {
 	exit 0;
 }
 
+my ($session, $error);
+
+# SNMP needed for both config and fetch.
+$session = Munin::Plugin::SNMP->session();
+$session->translate([-nosuchinstance => 0]);
+
 if ($Munin::Plugin::me =~ /_if_(\d+)$/) {
     # Also accept leading 0 like (snmp_router_if_01 .. snmp_router_if_24)
     $iface = int $1;
+} elsif ($Munin::Plugin::me =~ /_if_([\w\@.-]+)$/) {
+    my $if_alias = $session->get_by_regex("1.3.6.1.2.1.31.1.1.1.18.", "^$1\$");
```

Comment:
Maybe you could assign this SNMP identifier to a variable with a descriptive name? This will make it probably easier for future readers to understand, what this line is supposed to do.

Reply:
ahh, agreed. I should have done that from the outset. Will fix.